server/routerlicious/packages/routerlicious-base/src/test/nexus/io.spec.ts

```diff
@@ -709,178 +680,248 @@ describe("Routerlicious", () => {
-												referenceSequenceNumber: "invalid",
 											};
 
-											sendAndReturnExpectedSignal(clients[0], targetedSignal);
+											const expectedSignal = sendAndReturnExpectedSignal(
+												clients[1],
+												targetedSignal,
+											);
 
-											checkNack(clients[0], "Invalid signal message");
+											verifyExpectedClientSignals(clients, [expectedSignal]);
 										});
 
-										it("nacks signal that is not an array", () => {
-											const targetedSignal = {
-												targetClientId: clients[1].clientId,
+										it("can target a v1 client from a v2 client", () => {
+											const targetedSignal: ISentSignalMessage = {
+												targetClientId: clients[0].clientId,
 												content: stringSignalContent,
 											};
 
-											clients[0].socket.send(
-												"submitSignal",
-												clients[0].clientId,
+											const expectedSignal = sendAndReturnExpectedSignal(
+												clients[1],
 												targetedSignal,
 											);
-											checkNack(clients[0], "Invalid signal message");
-										});
-									});
-								} else if (description === "with v1 and v2 clients") {
-									it("should broadcast signal from a v1 client to all connected clients", () => {
-										const expectedSignal = sendValidAndReturnExpectedSignal(
-											clients[0],
-											stringSignalContent,
-										);
-
-										verifyExpectedClientSignals(clients, [expectedSignal]);
-									});
 
-									it("should broadcast signal from a v2 client to all connected clients", () => {
-										const targetedSignal: ISentSignalMessage = {
-											content: stringSignalContent,
-										};
-
-										const expectedSignal = sendAndReturnExpectedSignal(
-											clients[1],
-											targetedSignal,
-										);
+											verifyExpectedClientSignals(
+												[clients[0]],
+												[expectedSignal],
+											);
+											verifyExpectedClientSignals(
+												clients.filter((client) => client !== clients[0]),
+												[],
+											);
+										});
+									}
+								});
+								describe("sending multiple signals", () => {
+									[0, 1].forEach((clientIndex) => {
+										it("should broadcast signals sent from multiple clients to all connected clients", () => {
+											const firstSignal = sendValidAndReturnExpectedSignal(
+												clients[clientIndex],
+												"first signal",
+											);
+											const secondSignal = sendValidAndReturnExpectedSignal(
+												clients[clientIndex ^ 1],
+												"second signal",
+											);
 
-										verifyExpectedClientSignals(clients, [expectedSignal]);
+											verifyExpectedClientSignals(clients, [
+												firstSignal,
+												secondSignal,
+											]);
+										});
 									});
+									if (description === "with v1 clients") {
+										it(`should broadcast multiple batched signals sent from single clients to all connected clients`, () => {
```

Comment:
What about v2 client batching?

Reply:
I think if you refresh this is updated now

Reply:
I see it - nit: just missing some "s"s for plurality.